src/components/tag-input/tag-input.spec.ts

```diff
@@ -126,4 +126,13 @@ describe('Tag Input Component', () => {
         subscription.unsubscribe();
     });
 
+    it('should have an unique id for the input element', () => {
+        fixture.detectChanges();
+
+        const inputId = component.tagInput.nativeElement.id;
+        const elementsWithId = document.querySelectorAll(`#${inputId}`);
+
+        expect(elementsWithId.length).toBe(1);
```

Comment:
For future reference, this is the sort of thing I would expect as a unit test in this scenario:
```ts
component.tagInput.id = "test-id";
fixture.detectChanges();
const inputElement = nativeElement.querySelector<HTMLInputElement>('input.ux-tag-input');
expect(inputElement.id).toBe('test-id-input');
```

This test is very precise about what to expect as output for a given input, which means it will catch more unintended changes. For example, if we changed the id suffix to "-morb" your test would still pass, but it would be a breaking change since automated testing often depends on our IDs being predictable.